### ISSUE TITLE
Mark Respective Books as Freely Redistributable

### ISSUE THREAD

**Issue by SethFalco** [EbookFoundation/free-programming-books#3854]
Many of the books in the repository are free under various terms, or under terms which aren't made clear on the download page specifically.

It might be nice if for books released under open licenses like [Creative Commons](https://creativecommons.org/) or [Free Documentation License](https://www.gnu.org/licenses/fdl-1.3.html), that they be marked somehow in the repository.

This would be great to denote books which aren't only free, but also freely available to redistribute.

For example, there have been instances where a book or resource was previously free, but is no longer available for free. Many resources in the repository which aren't under an open license may no longer be available at any given moment for new users.

This would make it so it's clearer which books are certainly always going to be free, if not by the first party than by anyone that had already obtained the book under said license and has the right to continue redistributing it, vs books which must be downloaded from the website and  comes with no guarantee it'll remain freely available.

**Comment by eshellman** [comment]
good idea but maybe too much work. feel free to propose an implementation strategy

**Comment by kaeleyworks** [comment]
okay i am completely new to all of this. but im super interested and eager to learn and when i am taught something ive sought out i absorb it like a sponge. i need to know where to start. like literally pretend i ended up here on accident and know nothing. i can self teach pretty well, i just need to have a source for information and a starting point for reference. please someone, i promise i will surprise you. just a little time anf energy and i swear it wont be wasted. (payitforwards?)

**Comment by SethFalco** [comment]
@kaeleyworks 

I can see you're interested in [freeCodeCamp](https://github.com/freeCodeCamp/freeCodeCamp), so asking in the forum would be more appropriate and yield much better answers.

https://forum.freecodecamp.org/

I don't think this issue is the best place to hold the conversation, plus a subjective question like this is best heard by many. Regardless, I can note an _opinionated_, summary!
* [freeCodeCamp](https://www.freecodecamp.org/) if you like interactive tutorials, plus you'll get certificates.
  * They also have video courses on their YouTube channel. ([YouTube](https://www.youtube.com/freecodecamp))
* Books like any of the ones in [this repository](https://github.com/EbookFoundation/free-programming-books).
  * I'd especially recommend the books from Allen Downey if those topics interest you. ([Green Tea Press](https://greenteapress.com/wp/))
* Documentation of the tools or libraries you use, especially resources like [MDN Web Docs](https://developer.mozilla.org/en-US/).

The best way to learn will be the path where you have the most fun. Usually learning isn't much of a problem, staying motivated can be tough, though.

**Comment by Miladxsar23** [comment]
salam

**Comment by chirag-cyber** [comment]
Hello sir, please assign this task to me!

**Comment by akshatvermavi** [comment]
Bookstore events are an exciting part of the book freely redistribution. Many authors look forward to readings, signings, and bookstore-hosted workshops and have visions of hundreds of people waiting in line to buy their book. Here’s how to approach planning a bookstore event and some of the related pros and cons.
